src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl.py

```diff
@@ -678,6 +678,10 @@ def check_inputs(
                 raise ValueError(
                     f"`ip_adapter_image_embeds` has to be of type `list` but is {type(ip_adapter_image_embeds)}"
                 )
+            elif not hasattr(ip_adapter_image_embeds[0], "ndim"):
```

Comment:
```suggestion
            elif not isinstance(ip_adapter_image_embeds[0], torch.Tensor):
```
Both torch.Tensor and numpy.ndarray have the ndim attribute. If an user passes a list of numpy.ndarray objects as `ip_adapter_image_embeds`, he will get an error in the `prepare_ip_adapter_image_embeds` method, because this object does not have a chunk method.

Reply:
Sounds good to me 👍